docs/tutorials/hpa-recommendation.md

```diff
@@ -0,0 +1,219 @@
+    message: Analytics is ready
+    reason: AnalyticsReady
+    status: "True"
+    type: Ready
+  lastUpdateTime: "2022-05-15T13:34:19Z"
+  recommendations:
+  - lastStartTime: "2022-05-15T13:34:19Z"
+    message: Success
+    name: nginx-hpa-hpa-cd86s
+    namespace: default
+    targetRef:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: default
+    uid: b3cea8cb-259d-4cb2-bbbe-cd0e6544daaf
+```
+
+## Recommendation: Analytics result
+
+The recommendation name presents on `status.recommendations[0].name`. Then you can get recommendation detail by running:
+
+```bash
+kubectl get recommend nginx-hpa-hpa-cd86s -o yaml
+```
+
+The output is similar to:
+
+```yaml
+apiVersion: analysis.crane.io/v1alpha1
+kind: Recommendation
+metadata:
+  creationTimestamp: "2022-05-15T13:34:19Z"
+  generateName: nginx-hpa-hpa-
+  generation: 2
+  labels:
+    analysis.crane.io/analytics-name: nginx-hpa
+    analysis.crane.io/analytics-type: HPA
+    analysis.crane.io/analytics-uid: 5564edd0-d7cd-4da6-865b-27fa4fddf7c4
+    app: nginx
+  name: nginx-hpa-hpa-cd86s
+  namespace: default
+  ownerReferences:
+  - apiVersion: analysis.crane.io/v1alpha1
+    blockOwnerDeletion: false
+    controller: false
+    kind: Analytics
+    name: nginx-hpa
+    uid: 5564edd0-d7cd-4da6-865b-27fa4fddf7c4
+spec:
+  adoptionType: StatusAndAnnotation
+  completionStrategy:
+    completionStrategyType: Once
+  targetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+    namespace: default
+  type: HPA
+status:
+  conditions:
+  - lastTransitionTime: "2022-05-15T13:34:19Z"
+    message: Recommendation is ready
+    reason: RecommendationReady
+    status: "True"
+    type: Ready
+  lastUpdateTime: "2022-05-15T13:34:19Z"
+  recommendedValue: |
+    maxReplicas: 2
+    metrics:
+    - resource:
+        name: cpu
+        target:
+          averageUtilization: 75
+          type: Utilization
+      type: Resource
+    minReplicas: 2
+```
+
+## HPA Recommendation Algorithm model
+
+### Inspecting
+
+1. Workload with low replicas: If the replicas is too low,  may not be suitable for hpa recommendation. Associated configuration: ehpa.deployment-min-replicas | ehpa.statefulset-min-replicas | ehpa.workload-min-replicas
+2. Workload with a certain percentage of not running pods: if the workload of Pod mostly can't run normally, may not be suitable for flexibility. Associated configuration: ehpa.pod-min-ready-seconds | ehpa.pod-available-ratio
+3. Workload with low cpu usage: The low CPU usage workload means that there is no load pressure. In this case, we can't estimate it. Associated configuration: ehpa.min-cpu-usage-threshold
+4. Workload with low fluctuation of cpu usage: dividing of the maximum and minimum usage is defined as the fluctuation rate. If the fluctuation rate is too low, the workload will not benefit much from hpa. Associated configuration: ehpa.fluctuation-threshold 
```

Comment:
Make the key parameters in different styles.

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/35299017/168597176-2079ccef-f924-4eea-a257-c90c928a04b1.png">

```suggestion
1. Workload with low replicas: If the replicas is too low,  may not be suitable for hpa recommendation. Associated configuration: `ehpa.deployment-min-replicas` | `ehpa.statefulset-min-replicas` | `ehpa.workload-min-replicas`
2. Workload with a certain percentage of not running pods: if the workload of Pod mostly can't run normally, may not be suitable for flexibility. Associated configuration: `ehpa.pod-min-ready-seconds` | `ehpa.pod-available-ratio`
3. Workload with low CPU usage: The low CPU usage workload means that there is no load pressure. In this case, we can't estimate it. Associated configuration: `ehpa.min-cpu-usage-threshold`
4. Workload with low fluctuation of CPU usage: dividing of the maximum and minimum usage is defined as the fluctuation rate. If the fluctuation rate is too low, the workload will not benefit much from hpa. Associated configuration: `ehpa.fluctuation-threshold` 
```